recipe/meta.yaml

```diff
@@ -34,26 +29,22 @@ requirements:
     - setuptools
     - wheel
     - numpy
-    - dal-devel ==2021.6.0=hdb19cb5_916  # [linux]
-    - dal-devel ==2021.6.0=haf03e11_929  # [osx]
-    - dal-devel ==2021.6.0=h59b6b97_874  # [win]
+    - dal-devel 2023.0.1
     - cython
     - jinja2
     - mpich      # [not win]
     - clang-tools
     - pybind11
   run:
     - python
-    - dal ==2021.6.0=hdb19cb5_916  # [linux]
-    - dal ==2021.6.0=haf03e11_929  # [osx]
-    - dal ==2021.6.0=h59b6b97_874  # [win]
+    - dal 2023.0.1
```

Comment:
Missing `scikit-learn >=0.24` https://github.com/intel/scikit-learn-intelex/blob/ef80ea962b05e6172df600801d883c454db84ba1/setup.py#L426. And then we can remove it from test/requires. Or that package only for tests are needed?

Reply:
I believe it's just a test requirement. Checked [Intel's channel](https://anaconda.org/intel/daal4py/files) for `daal4py` and they do not include `scikit-learn` as a run dep.